### ISSUE TITLE
source file attribute gets changed

### ISSUE THREAD

**Issue by sorah** [k0kubun/mitamae#47]
```
[alice@srv ~]$ cat > recipe.rb
remote_file '/tmp/dest.rb' do
  owner 'root'
  group 'root'
  mode  '0440'
  source './src.rb'
end
[alice@srv ~]$ cat > src.rb
aa
```

```
[alice@srv ~]$ ls -la src.rb
-rw-r--r-- 1 alice alice 3 Jan  5 13:56 src.rb
[alice@srv ~]$ sudo /opt/mitamae/bin/mitamae local recipe.rb
 INFO : Starting MItamae...
 INFO : Recipe: /home/alice/recipe.rb
 INFO :   remote_file[/tmp/dest.rb] exist will change from 'false' to 'true'
 INFO :   remote_file[/tmp/dest.rb] mode will be '0440'
 INFO :   remote_file[/tmp/dest.rb] owner will be 'root'
 INFO :   remote_file[/tmp/dest.rb] group will be 'root'
 INFO :   diff:
 INFO :   --- /dev/null 2016-12-20 21:33:33.850000000 +0000
 INFO :   +++ /home/alice/src.rb      2017-01-05 13:56:31.851855466 +0000
 INFO :   @@ -0,0 +1 @@
 INFO :   +aa
[alice@srv ~]$ ls -la src.rb
-r--r----- 1 root root 3 Jan  5 13:56 src.rb
[alice@srv ~]$ /opt/mitamae/bin/mitamae version
MItamae v1.3.2
```

**Comment by eagletmt** [comment]
It seems to be caused by #41 :bow:

**Comment by eagletmt** [comment]
Fixed in v1.3.3
